modules/nf-scil/betcrop/fslbetcrop/main.nf

```diff
@@ -64,6 +86,7 @@ process BETCROP_FSLBETCROP {
 
     touch ${prefix}__dwi_bet_cropped.nii.gz
     touch ${prefix}__dwi_bet_cropped_mask.nii.gz
+    touch ${prefix}__dwi_boundingBox.pkl
```

Comment:
I think for documentation and clarity it would be best to change the `_dwi_` for `_image_` since the module can accept a t1 also.